ixmp4/data/api/optimization/column.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+from typing import ClassVar
+
+from ixmp4.data.abstract import optimization as abstract
+
+from .. import base
+from .indexset import IndexSet
+
+
+class Column(base.BaseModel):
+    NotFound: ClassVar = abstract.Column.NotFound
+    NotUnique: ClassVar = abstract.Column.NotUnique
+    DeletionPrevented: ClassVar = abstract.Column.DeletionPrevented
+
+    id: int
+    name: str
+    dtype: str
+    table__id: int
+    indexset: IndexSet
+    constrained_to_indexset: int
+    unique: bool
+
+    created_at: datetime | None
```

Comment:
If the columns are created at the same time as the Table, I don't think that we need to have the created-info.